Bug: 插件无法正常加载

### Describe the bug

从插件市场下载安装后无法正常加载。

### Steps to reproduce

如上

### Expected behavior

正常加载和启用

### Screenshots

2023-04-11 13:42:22 [W] app Error: Cannot find module './locales/zh-cn'
                        Require stack:
                        - /home/koishi-app/node_modules/koishi-plugin-booru/lib/index.js
                        - /home/koishi-app/node_modules/@koishijs/loader/lib/index.js
                        - /home/koishi-app/node_modules/koishi/lib/worker/index.js
                            at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
                            at Function.Module._load (node:internal/modules/cjs/loader:778:27)
                            at Module.require (node:internal/modules/cjs/loader:1005:19)
                            at require (node:internal/modules/cjs/helpers:102:18)
                            at Object.apply (/home/koishi-app/node_modules/koishi-plugin-booru/lib/index.js:111:27)
                            at /home/koishi-app/node_modules/cordis/lib/index.cjs:415:40
                            at MainScope.ensure (/home/koishi-app/node_modules/cordis/lib/index.cjs:257:18)
                            at MainScope.apply (/home/koishi-app/node_modules/cordis/lib/index.cjs:415:14)
                            at MainScope.start (/home/koishi-app/node_modules/cordis/lib/index.cjs:475:12)
                            at MainScope.restart (/home/koishi-app/node_modules/cordis/lib/index.cjs:235:10)

### Relevant log output

```shell
2023-04-11 13:42:22 [W] app Error: Cannot find module './locales/zh-cn'
                        Require stack:
                        - /home/koishi-app/node_modules/koishi-plugin-booru/lib/index.js
                        - /home/koishi-app/node_modules/@koishijs/loader/lib/index.js
                        - /home/koishi-app/node_modules/koishi/lib/worker/index.js
                            at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
                            at Function.Module._load (node:internal/modules/cjs/loader:778:27)
                            at Module.require (node:internal/modules/cjs/loader:1005:19)
                            at require (node:internal/modules/cjs/helpers:102:18)
                            at Object.apply (/home/koishi-app/node_modules/koishi-plugin-booru/lib/index.js:111:27)
                            at /home/koishi-app/node_modules/cordis/lib/index.cjs:415:40
                            at MainScope.ensure (/home/koishi-app/node_modules/cordis/lib/index.cjs:257:18)
                            at MainScope.apply (/home/koishi-app/node_modules/cordis/lib/index.cjs:415:14)
                            at MainScope.start (/home/koishi-app/node_modules/cordis/lib/index.cjs:475:12)
                            at MainScope.restart (/home/koishi-app/node_modules/cordis/lib/index.cjs:235:10)
```


### Launcher

Koishi Android

### Sources

Danbooru, Gelbooru, Konachan.com (NSFW), Konachan.net (SFW), Lolibooru, Lolicon, Pixiv, Safebooru, SankakuComplex, Yande.re

### Versions

- OS: <!-- e.g. Windows 10 -->Android 13
- Adapter: <!-- e.g. Discord --> adapter-onebot
- Node version: <!-- e.g. 18.12.0 -->unknown，安卓版本不清楚如何查看
- Koishi version: <!-- e.g. 4.9.9 -->4.12.4


### Additional context

_No response_

--- a/packages/core/src/index.ts
+++ b/packages/core/src/index.ts
@@ -119,7 +119,7 @@ export const Config = Schema.intersect([
 export function apply(ctx: Context, config: Config) {
   ctx.plugin(ImageService, config)
 
-  ctx.i18n.define('zh', require('./locales/zh-cn'))
+  ctx.i18n.define('zh', require('./locales/zh-CN'))
 
   const count = (value: string, session: Session) => {
     const count = parseInt(value)

